code/ttgo_tests/TTGO_Sender_Receiver/TTGO_Sender_Receiver.ino

```diff
@@ -0,0 +1,98 @@
+#include <LoRa.h> //https://github.com/sandeepmistry/arduino-LoRa
+#include <SPI.h>  
+#include <Wire.h>
+#include "SSD1306.h" 
+
+#define SCK     5    // GPIO5  -- SX1278's SCK
+#define MISO    19   // GPIO19 -- SX1278's MISO
+#define MOSI    27   // GPIO27 -- SX1278's MOSI
+#define SS      18   // GPIO18 -- SX1278's CS
+#define RST     14   // GPIO14 -- SX1278's RESET
+#define DI0     26   // GPIO26 -- SX1278's IRQ(Interrupt Request)
+#define BAND    433E6// 433 MHz Freq
+
+SSD1306 display(0x3c, 4, 15);
+String rssi ;
+String packSize ;
+String packet ;
+String sentPacket ;
+
+void displayLoraData(){
+  //Prints various statistics and data to OLED
+  display.clear();
+  display.drawString(50 , 0 , "Bytes: " + packSize);
+  display.drawStringMaxWidth(0 , 15 , 128, "RX:  " + packet);
+  display.drawString(0, 0, "RSSI: "+ rssi);  
+  display.display();
+}
+
+void readPacket(int packetSize) {
+  //Reads packet from radio socket
+  packet ="";
+  packSize = String(packetSize,DEC);
+  for (int i = 0; i < packetSize; i++) {
```

Comment:
reading by one byte makes no sense to me when `LoRa` object provides [`steams` capability](https://www.arduino.cc/reference/en/language/functions/communication/stream/)

Reply:
I have changed this to LoRa.readString() (a Stream property which reads streamed data into a string, compiles correctly), have no idea if it works though as i'm currently not in possession of the test board.

Reply:
I guess that will do